lib/json5.js

```diff
@@ -43,6 +43,7 @@ JSON5.parse = (function () {
             '\xA0',
             '\uFEFF'
         ],
+        operator = ['+', '-', '*', '/'],
```

Comment:
Why not support all operators which could legitimately operate on numeric constants - therefore including !, ^, &, &&, |, and ||. I think there's no reason to exclude boolean constants from this work - therefore supporting true/false in addition to the numbers which have been already supported.